vite.config.ts

```diff
@@ -0,0 +1,53 @@
+import react from '@vitejs/plugin-react';
+import tailwindcss from 'tailwindcss';
+import {defineConfig, loadEnv} from 'vite';
+import viteTsconfigPaths from 'vite-tsconfig-paths';
+import { VitePWA } from 'vite-plugin-pwa';
```

Comment:
That extra space should have been caught by the linters, but we currently only lint `src/`. We could add this file to the config as well:

```json
{
      "lint": "eslint --ext js,jsx,ts,tsx src/ && prettier --check src/",
}
```

Reply:
Extended to also check all other `.js`, `.ts` and `.cjs` files that are at the root of the project